File: OLD Repos/Linear Algebra/Determinant/N-ary Arrangement.jl
#### N-ary Arrangement

# Input a Vector as Sequence:
A = [2,4,3,1]
println("The sequence is ",A)
# Print the elements number of the Vector:
println("The number of elements is ",length(A))
# Initialize the Order Number:
o = Int64(0)
# Initialize the Reverse Order Number:
t = Int64(0)

# Calculus the Order Number and the Reverse Order Number:
for i in 1:length(A)
    for j in 1:(i-1)
        # print(i,",",j,";")
        if A[j] < A[i]
            global o = o + 1
        elseif A[j] > A[i]
            global t = t + 1
        end
    end
end
println("The Order Number of this Sequence is: ", o)
println("The Reverse Order Number of this Sequence is: ", t)

# Determine whether the sequence is odd or even:
if rem(0,2) == 0
    println("This Sequence is even.")
elseif  rem(0,1) == 0
    println("This Sequence is odd.")
end

File: OLD Repos/Linear Algebra/Determinant/N-ary Arrangement.py
#### N-ary Arrangement

import numpy as np

# Input a Vector as Sequence:
A = np.array([2,4,3,1])
print("The sequence is ",A)
# Print the elements number of the Vector:
print("The number of elements is ",A.shape[0])

# Initialize the Order Number:
o = int(0)
# Initialize the Reverse Order Number:
t = int(0)

# Calculus the Order Number and the Reverse Order Number:
for i in np.arange(0,A.shape[0]):
    for j in np.arange(0,i):
        # print(i,j)
        if A[j] < A[i]:
            o = o + 1
        elif A[j] > A[i]:
            t = t + 1
print("The Order Number of this Sequence is: ",o)
print("The Reverse Order Number of this Sequence is: ",t)

# Determine whether the sequence is odd or even:
if np.remainder(0,2) == 0:
    print("This Sequence is even.")
elif  np.remainder(0,1) == 0:
    print("This Sequence is odd.")